docs/UserGuide.md

```diff
@@ -42,6 +66,18 @@ Tuteez is a **desktop address book app designed specifically for tech-savvy priv
 
 --------------------------------------------------------------------------------------------------------------------
 
+## UI 
```

Comment:
I like the information you included, this will be useful for new and potential users!